sopel_modules/github/webhook.py

```diff
@@ -46,32 +67,16 @@ def setup_webhook(sopel):
     sopel.memory['gh_webhook_server'] = base
     sopel.memory['gh_webhook_thread'] = server
 
-    conn = sopel.db.connect()
-    c = conn.cursor()
 
+    engine = sopel.db.engine
```

Comment:
Uhhhh, this attribute Should™ be for Sopel's internal use only. The API-defined way of getting at this is [`sopel.db.connect()`](https://github.com/sopel-irc/sopel/blob/514d6b8c70b5853d96dd2093d87eecfd8b176c62/sopel/db.py#L177-L179), which also runs `.connect()` on the engine for you.

Reply:
This is what happens when I'm tired and writing code :D

Reply:
Done.